### ISSUE TITLE
Specifying alternate interface seems to not work

### ISSUE THREAD

**Issue by ColdHeat** [DigitalSecurity/btlejuice#6]
When I do `sudo btlejuice-proxy -i  hci1`

I get `Using interface hciNaN`

Any idea what is up with that?

**Comment by virtualabs** [comment]
Of course. The _-i_ parameter expects the interface number, not its full name. But yes, this should be mentioned and checked by the software.
